### PR TITLE
Angularjs

### PR DESCRIPTION
**Pull request by Pristavka**

### CODE REVIEW

File: angularjs/app/app.module.js [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
+import taskFormModule from './components/task-form/task-form.module';
+import taskListModule from './components/task-list/task-list.module';
+import editModule from './components/edit-form/edit-form.module';
+
+const AppModule = angular
+  .module('app', [
+    ngRoute,
+    ngSanitize,
+    headerModule,
+    taskFormModule,
```
Comment (NikitaSedyx) [inline angularjs/app/app.module.js]: why do you need module for each component like `taskFormModule` and `taskListModule`, imo you should have only one for tasks

File: angularjs/app/app.module.js [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
+    taskListModule,
+    editModule
+  ])
+  .service('TodoService', TodoService)
```
Comment (NikitaSedyx) [inline angularjs/app/app.module.js]: is it shared service?

File: angularjs/app/components/task-form/task-form.component.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
-    controllerAs: 'ctrl'
-  });
-})();
+import template from './task-form.template';
+import TaskFormController from './task-form.controller';
+
+const taskFormComponent = {
+  template,
+  controller: TaskFormController,
+  controllerAs: 'ctrl'
```
Comment (NikitaSedyx) [inline angularjs/app/components/task-form/task-form.component.js]: as you was told earlier it's better to use unique names

File: angularjs/app/app.module.js (hunk 2 above)
Comment (Pristavka) [reply angularjs/app/app.module.js]: fixed

File: angularjs/app/components/task-form/task-form.component.js (hunk 3 above)
Comment (Pristavka) [reply angularjs/app/components/task-form/task-form.component.js]: fixed

File: angularjs/app/app.module.js (hunk 1 above)
Comment (Pristavka) [reply angularjs/app/app.module.js]: fixed

**Pull request merged**